app/controllers/fines_controller.rb

```diff
@@ -7,13 +7,26 @@ class FinesController < ApplicationController
   def index
     @fines = fines
     @checkouts = checkouts
+    @pay_link = cybersource
```

Comment:
It feels a little funny to me to make this an instance variable on the controller; is it maybe more of a helper method?

Reply:
Creates a payments_helper module taht is called from the view.

Reply:
Moved this to new PaymentsController